packages/modeling/src/utils/padArrayToLength.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Fill an array with a default value, until it reaches a specified length.
+ * @param {Array} arr - the array to potentially add elements to.
+ * @param padding - the element to add to the array to reach the desired length.
```

Comment:
Hmm... anything is ok... how to say anything in JSDOC?

Reply:
{*} apparently